### PR TITLE
fix some links in es guidelines for authors and translators

### PR DESCRIPTION
**Pull request by arojascastro**
I noticed that there are some broken links in our es guidelines for authors. I tried to fix them all, but maybe someone else can double check and approve these changes? Gracias.

Matthew can you check that I did not make any mistake? Thanks a lot.

### CODE REVIEW

**Comment by acrymble** [comment]
@arojascastro you have a broken link here. You're using a relative link but you need the full link to the submissions repository.

**Comment by arojascastro** [comment]
Thank you!

File: es/guia-para-autores.md [hunk 1]
```diff
@@ -16,11 +16,11 @@ original: author-guidelines
 
 Si quieres traducir una lección, tienes una idea para una lección nueva o ya has escrito un tutorial que crees que puede adaptarse a *The Programming Historian en español*, contacta con [Antonio Rojas Castro]. Cuanto antes te pongas en contacto con nosotros, mucho mejor; de esta manera, te ayudaremos a plantear adecuadamente tu contribución, teniendo en cuenta el público objetivo y el nivel de conocimientos necesarios. También te asignaremos un editor para ayudarte a resolver dudas y a desarrollar la lección de la mejor manera.
 
-**¿Qué lecciones traducir?** Si quieres traducir una lección, por favor, revisa la lista de [traducciones pendientes] y ponte en contacto con nosotros. Buscamos traducciones rigurosas, de lectura amena y que, además, tengan en cuenta el contexto de España y América Latina y los recursos disponibles en lengua española.
```
Comment (mdlincoln) [inline es/guia-para-autores.md:19]: If you look at the bottom of this markdown file, I think the `[traducciones pendientes]` link is already specified there. You should not need to specify the url in parentheses after it. And the link seems to work already on the live site?

File: es/guia-para-autores.md [hunk 2]
```diff
@@ -16,11 +16,11 @@ original: author-guidelines
...
 
 **¿Qué tipo de lección queremos?** Aceptamos tutoriales relevantes para las humanidades, dirigidos a cualquier nivel de aptitud técnica y experiencia, que se centren en un problema o proceso, que puedan ser sostenibles a largo plazo y que estén dirigidos a una audiencia global. El alcance y la longitud del tutorial han de corresponderse con la complejidad de la tarea que se enseña. Los tutoriales no deben exceder las 8.000 palabras (incluyendo el código) sin el permiso explícito del editor y que se otorgará únicamente en circunstancias excepcionales. Esperamos que la mayoría de las lecciones tengan entre 4.000 y 6.000 palabras. Puede que pidamos dividir en varios tutoriales las lecciones más largas.
 
-**En resumen, aceptamos todo tipo de propuestas.** Consulta nuestras [lecciones publicadas], lee nuestras [directrices para revisores] o explora las [lecciones actualmente en desarrollo] (https://github.com/programminghistorian/ph-submissions/tree/gh-pages/lessons) para hacerte una mejor idea de lo que publicamos. Animamos al envío de propuestas de lecciones sobre temas ya cubiertos o en desarrollo, siempre que la lección nueva haga una contribución propia.
```
Comment (mdlincoln) [inline es/guia-para-autores.md:23]: Again, I think the links for most of these pages are already specified at the bottom of this markdown file. If they need any corrections, you should make the corrections there.

**Review by mdlincoln — CHANGES_REQUESTED** [review]
Most of these links in this file are using the "reference" style of markdown link. Which works like this:

```md
I'm linking to [GitHub]

[GitHub]: https://github.com
```

So you should only need to modify the references found at the bottom of the file.

File: es/guia-para-autores.md (hunk 1 above)
Comment (arojascastro) [reply es/guia-para-autores.md:19]: Ok, I'll fix them there. Sorry.

**Comment by mdlincoln** [comment]
Note @arojascastro I suggested some changes to this PR.

File: es/guia-para-autores.md (hunk 2 above)
Comment (arojascastro) [reply es/guia-para-autores.md:23]: Ok, thanks. Can we reject this pull request to revert my changes? if so I will fix the links as you suggest, I thin it is easier.

Comment (mdlincoln) [reply es/guia-para-autores.md:23]: OK!

**Comment by mdlincoln** [comment]
@arojascastro will resubmit this PR: https://github.com/programminghistorian/jekyll/pull/900#discussion_r196329492

**Pull request closed without merging**